### ISSUE TITLE
Cannot save parameter with value = 0

### ISSUE THREAD

**Issue by laurabennett** [spicywebau/craft-embedded-assets#89]
I want to add rel=0 to YouTube embed code so that it will only show related videos from my channel at the end of a video.  When I try to save it give me an error, Parameter key `value` is required.

**Comment by timprint** [comment]
I was just trying to do the same thing. Is there another way to do this?

**Comment by alexagui** [comment]
I didn't get an error saving the value but even though I added `rel=0` in the control panel settings it's ignored on the front-end.

**Comment by laurabennett** [comment]
Why was this closed it's still not working.

**Comment by ttempleton** [comment]
The settings validation error has now been fixed in Embedded Assets 2.5.1, but you won't be able to use it to set `rel=0` as YouTube will ignore it, see #22.  Instead, you could set it using the macro provided in that issue, or the since-added `getVideoCode()` method:

```twig
{% set vid = craft.embeddedAssets.get(entry.vid.one()) %}
{{ vid.getVideoCode(['rel=0']) }}
```
